src/devhub/components/island/support.jsx

```diff
@@ -23,8 +23,8 @@ const Items = [
       </>
     ),
     cta: {
-      href: "/devhub.near/widget/app?page=community&handle=education&tab=Wiki%202",
-      title: "Book a meeting ↗",
+      href: "/devhub.near/widget/app?page=community&handle=devrel&tab=Office%20hours%20/%20Support",
```

Comment:
Instead of "Office%20hours%20/%20Support", we do:
"office-hours-support" and in community page, when we do the check, we also normalize before checking. 

There is an existing normalize function we can reuse [here](https://github.com/NEAR-DevHub/neardevhub-bos/blob/e546fa07e13b1d00e29bd3438e91a0ce1d5e52c9/src/devhub/page/create.jsx#L197)